tooling/dep2.js

```diff
@@ -0,0 +1,22 @@
+
+const fs = require('fs');
+
+const glob = require('glob');
+
+// gets reference to all the package.json file
+glob('packages/**/package.json', {}, (er, files) => {
+  files.forEach((file) => {
+    const currentPackageJson = fs.readFileSync(file);
+
+    const parsedPackages = JSON.parse(currentPackageJson);
+
+    let stringified = JSON.stringify(parsedPackages, null, 2);
```

Comment:
It would be safer to parse the JSON than update the version numbers in the object and stringify it only before you save it